src/static/js/event_find_results.js

```diff
@@ -0,0 +1,214 @@
+        ( "EventCountry", _("Country", l) ),
+*/
+            let labels = {
+                "CreatedBy": _("Created By"),
+                "CreatedDate": _("Created Date"),
+                "LastChangedBy":  _("Last Changed By"),
+                "LastChangedDate":  _("Last Change Date"),
+                "EventName":  _("Event Name"),
+                "StartDateTime":  _("Start Date"),
+                "EndDateTime":  _("End Date"),
+                "EventOwnerName":  _("Location"),
+                "EventAddress":  _("Address"),
+                "EventTown":  _("City"),
+                "EventCounty":  _("County"),
+                "EventPostcode":  _("Zipcode"),
+                "EventCountry":  _("Country"),
+            };
+            if (labels.hasOwnProperty(name)) {
+                return labels[name];
+            }
+            if (add) {
+                let addrow = common.get_row(add, name, "FIELDNAME");
+                if (addrow) { return addrow.FIELDLABEL; }
+            }
+            return name;
+        },
+        
+        /**
+         * Returns a formatted column
+         * row: The event resultset row
+         * name: The name of the column
+         * value: The value of the row/column to format from the resultset
+         * add: The additional row results
+         */
+        format_column: function(row, name, value, add) {
+            const DATE_FIELDS = [ "StartDateTime", "EndDateTime" ],
+            DATETIME_FIELDS = [ "CreatedDate", "LastChangedDate" ],
+            STRING_FIELDS = [ "EventName", "EventOwnerName", "EventAddress", "EventTown", "EventCounty", "EventPostcode", "EventCountry" ];
+            let rv = "";
+            if ($.inArray(name, DATETIME_FIELDS) > -1) {
+              rv = format.date(value);
+              if (format.time(value) != "00:00:00") {
+                  rv += " " + format.time(value);
+              }            
+            }
+            else if ($.inArray(name, DATE_FIELDS) > -1) {
+              rv = format.date(value);
+            }
+            else if ($.inArray(name, STRING_FIELDS) > -1) {
+                rv = common.nulltostr(value);
+            }
+            else if (add) {
+                $.each(add, function(i, v) {
+                    if (v.LINKID == row.ID && v.FIELDNAME.toLowerCase() == name.toLowerCase()) {
+                        if (v.FIELDTYPE == additional.YESNO) { 
+                            rv = v.VALUE == "1" ? _("Yes") : _("No");
+                        }
+                        else if (v.FIELDTYPE == additional.MONEY) {
+                            rv = format.currency(v.VALUE);
+                        }
+                        else if (v.FIELDTYPE == additional.ANIMAL_LOOKUP) {
+                            rv = '<a href="animal?id=' + v.VALUE + '">' + v.ANIMALNAME + '</a>';
+                        }
+                        else if ([additional.PERSON_LOOKUP, additional.VET, additional.SPONSOR].includes(v.FIELDTYPE)) {
```

Comment:
@bobintetley you might want to change other *_results.js pages to use this instead of only additional.PERSON_LOOKUP

Reply:
I think I'd add a function to additional to check if it's a person type rather than copying the same test everywhere. Makes it easy to add another in future then.